app/controllers/maintenance_tasks/runs_controller.rb

```diff
@@ -34,6 +34,19 @@ def cancel
       redirect_to(task_path(@run.task_name), alert: error.message)
     end
 
+    # Downloads the CSV attachment associated with the Run.
+    def csv_file
+      respond_to do |format|
+        format.csv do
+          csv_file = @run.csv_file.download
+          task_name = @run.task_name.underscore.gsub('/', '_')
```

Comment:
Why not keep the task name as is? It looks like Firefox removes the `::` and puts a space, while Safari and Chrome use `__`.